src/resources/js/tickets.js

```diff
@@ -3,6 +3,18 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 (function( window, $, undefined ) {
 	'use strict';
 
+	var $event_pickers = $( document.getElementById( 'tribe-event-datepickers' ) ),
+		$tribe_tickets = $( document.getElementById( 'tribetickets' ) ),
+		$tickets_container = $( document.getElementById( 'event_tickets' ) ),
+		$enable_global_stock = $( document.getElementById( 'tribe-tickets-enable-global-stock' ) ),
+		$global_stock_level = $( document.getElementById( 'tribe-tickets-global-stock-level' ) ),
+		global_stock_setting_changed = false,
+		$body = $( 'html, body' ),
+		startofweek = 0,
+		$base_panel = $( document.getElementById( 'panel_base' ) ),
+		$edit_panel = $( document.getElementById( 'panel_edit' ) ),
+		$settings_panel = $( document.getElementById( 'panel_settings' ) );
```

Comment:
I find comma separated variables icky. I have added a note about why in our JS docs page: http://moderntribe.github.io/products-engineering/js/

My thoughts aren't a definitive standard, but something to consider.